addons/pf-maint.pl

```diff
@@ -40,6 +40,7 @@ =head1 SYNOPSIS
 use IO::Handle;
 use Term::ReadKey;
 use IO::Interactive qw(is_interactive);
+use LWP::Protocol::connect;
```

Comment:
I don't see this package in the spec and in the debian dependencies, is it an extra package or does it come with another one ?

Reply:
I think its the Fingerbank Perl lib that uses it so its installed on PF installs. Let me add it explicitly in the PF packaging in case we change the Fingerbank Perl packaging at some point

Reply:
I added that package in our spec, its already in our repo so that should be golden